src/electron/release_linux_action.sh

```diff
@@ -22,10 +22,14 @@ yarn do src/electron/package_common
 
 scripts/environment_json.sh -r -p linux > www/environment.json
 
+# Set-up electron to only download new updates but never upload new ones
 electron-builder \
   --linux \
+  # Disable publishing from Electron
```

Comment:
Maybe "Don't publish binary to S3"

"from Electron" is not quite clear. Electron doesn't publish anything.